Description: Fix the ActiveJobs loop in cupsdCheckJobs()
 When looping over the active jobs, if a pending job is started it will
 have looped through the ActiveJobs array, meaning further pending
 active jobs will be skipped for another 10 seconds (the senddoc
 delay).
Author: Tim Waugh <twaugh@redhat.com>
Origin: upstream
Bug: https://cups.org/str.php?L4646

--- a/scheduler/job.c
+++ b/scheduler/job.c
@@ -392,7 +392,9 @@
 	  * Start the job...
 	  */
 
+	  cupsArraySave(ActiveJobs);
 	  start_job(job, printer);
+	  cupsArrayRestore(ActiveJobs);
 	}
       }
     }
